be-rust/src/commonMain/kotlin/lang/temper/be/rust/RustBackend.kt

```diff
@@ -159,20 +164,28 @@ class RustBackend(setup: BackendSetup<RustBackend>) : Backend<RustBackend>(Facto
                 this["name"] = names.packageNaming.packageName
                 this["version"] = libraryConfiguration.versionOrDefault()
                 libraryConfiguration.license()?.let { this["license"] = it }
-            }.entries.joinToString("\n") { (key, value) ->
-                "$key = ${jsonEscaper.escape(value)}"
+                libraryConfiguration.description()?.let { this["description"] = it }
+                libraryConfiguration.homepage()?.let { this["homepage"] = it }
+                libraryConfiguration.repository()?.let { this["repository"] = it }
+            }.map { (key, value) ->
+                // Top-level string escape all above values.
+                key to jsonEscaper.escape(value)
+            }.toMap() + buildMap {
+                // *Not* top-level string values here.
+                libraryConfiguration.authors()?.let { this["authors"] = "[${jsonEscaper.escape(it)}]" }
             }
```

Comment:
The above remains rather inefficient, but at least this part isn't typically in a tight loop.